components/epd_driver/i2s_data_bus.c

```diff
@@ -162,8 +162,9 @@ void i2s_bus_init(i2s_bus_config *cfg) {
   // (Smallest possible divider according to the spec).
   dev->sample_rate_conf.tx_bck_div_num = 2;
 
-  // Initialize Audio Clock (APLL) for 80 Mhz.
-  rtc_clk_apll_enable(1, 0, 0, 8, 1);
```

Comment:
As long as we do not know exactly why this workaround works, I'd rather want to keep it at the frequency specified in the data sheet, at least on the main branch.

Reply:
True, added this into menuconfig as new display type.
This is only way to drive such display for me.